### PR TITLE
fix: Small stupid misstake in GroupDetails menu (AR-2652)

### PR DESCRIPTION
**Pull request by borichellow**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-2652" title="AR-2652" target="_blank">BugAR-2652</a>  Delete and Leave Group options are not displayed anymore
  </td></table>
  <br />
 

# What's new in this PR?

### Issues

mistake that causes wrong menu in GroupDetails screen

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-1112/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4774139)(4774139)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4774139)(4774139)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-1112/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4774337)(4774337)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4774337)(4774337)-dev-release.apk)

File: app/src/test/kotlin/com/wire/android/ui/home/conversations/details/GroupConversationDetailsViewModelTest.kt [hunk 1]
```diff
@@ -367,10 +370,34 @@ class GroupConversationDetailsViewModelTest {
...
+        // When - Then
+        assertEquals(expected, viewModel.conversationSheetContent)
+    }
+
     companion object {
+        val dummyConversationId = ConversationId("some-dummy-value","some.dummy.domain")
         val testGroup = ConversationDetails.Group(
             Conversation(
-                id = ConversationId("conv_id", "domain"),
+                id = dummyConversationId,
```
Comment (gongracr) [inline app/src/test/kotlin/com/wire/android/ui/home/conversations/details/GroupConversationDetailsViewModelTest.kt:400]: Ideally we should provide this parameter on demand on each test:
     `.withConversationId(expectedConversationId)`
That way, we could encapsulate the initial parameters feeding to each test independently.

**Review by gongracr — APPROVED** [review]
Small human mistake but great that it is covered now with unit tests 🤖 🦺 
I just left a small improvement comment but nothing blocker.

**Pull request merged**